scripts/migration/organization-mapping.ts

```diff
@@ -0,0 +1,168 @@
+/**
+ * Migration mapping for MobX-RESTful-migrator
+ * Converts existing data transformation logic to migration schema format
+ */
+
+import { MigrationSchema } from 'mobx-restful-migrator';
+import { OrganizationData, Organization, ExtendedUserData } from '../types';
+
+import { AddressTransformer } from '../transformers/address-transformer';
+import { DateTransformer } from '../transformers/date-transformer';
+import { ServiceTransformer } from '../transformers/service-transformer';
+import { QualificationTransformer } from '../transformers/qualification-transformer';
+import { UserTransformer } from '../transformers/user-transformer';
+import { DataUtils } from '../utils/data-utils';
+
+import { UserModel, configureModels } from '../models/strapi-models';
+
+/**
+ * Create migration mapping configuration
+ * This translates the existing DataTransformer logic into MobX-RESTful-migrator format
+ */
+export function createMigrationMapping(
+  baseURL: string,
+  token: string,
+): MigrationSchema<Organization, OrganizationData> {
+  // Configure models with connection info
+  configureModels(baseURL, token);
+
+  return {
+    // Simple 1-to-1 mappings using field names
+    常用名称: ({ ['常用名称']: value, name }) => ({
+      name: { value: value || name || '' },
```

Comment:
```suggestion
export const migrationMapping: MigrationSchema<Organization, OrganizationData> = {
    常用名称: ({ 常用名称: value }) => ({
      name: { value, unique: true },
```
把 `OrganizationData` 拆成两个类型，源头都是中文字段，目标都是英文字段。